CHANGES.md

```diff
@@ -26,6 +26,10 @@ Format of the entries must be.
 
 * A cluster's IP detect script may be changed with a config upgrade (DCOS_OSS-2389)
 
+* DC/OS Net: Support Mesos Windows Agent (DCOS_OSS-2073)
```

Comment:
@urbanserj  - Could you please verify if this changes.md addition is good for your changes you are bringing in with https://github.com/dcos/dcos/pull/2635/ ?

In future, you could add these CHANGES.md addition in your PR too.